public/js/monitor.js

```diff
@@ -436,6 +489,50 @@ function resizeDashboardMargin() {
   }
 
   const dropDownMenu = document.querySelector(".mobile-nav.show-mobile");
+  const submitBtn = document.querySelector("#scan-user-email input[type='submit']");
+
   dropDownMenu.addEventListener("click", () => toggleDropDownMenu(dropDownMenu));
 
+  const acceptedFirstLocalesIncludesEnglish =
+  (acceptedLanguages[0].includes("en-US") || acceptedLanguages[0].includes("en"));
```

Comment:
Is `acceptedLanguages[0]` a string? If so, `.includes("en")` will return `true` for any `en` language, so, the `en-US`-specific check isn't needed?

Reply:
acceptedLanguages[0] is a string. Revised!